conversions/c_atoi_str_to_integer.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
```

Comment:
please add documentation to your code in [doxygen](https://doxygen.nl) standards. You can also refer to [this file](https://github.com/kvedala/C/blob/master/numerical_methods/realtime_stats.c)

Reply:
Done

Reply:
The author name should be yours, dont you think 😄